src/data/constants/index.js

```diff
@@ -13,6 +13,9 @@ const IN_REVIEW_STATUS = [REVIEW_BY_LEGAL, REVIEW_BY_INTERNAL];
 const POST_REVIEW_STATUSES = [PUBLISHED, REVIEWED];
 
 const DATE_FORMAT = 'YYYY-MM-DD[T]HH:mm:ss[Z]';
+const DATE_INPUT_GUARD = '20[1-9][0-9]/(0[1-9]|1[012])/(0[1-9]|[12][0-9]|3[01])';
```

Comment:
I think this variable name would make more sense as `DATE_PATTERN` since everywhere it is used, you are passing it in as the pattern prop.